client/my-sites/customer-home/cards/tasks/growth-summit/index.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Task from 'my-sites/customer-home/cards/tasks/task';
+import { TASK_GROWTH_SUMMIT } from 'my-sites/customer-home/cards/constants';
+import growthSummitIllustration from 'assets/images/customer-home/illustration--growth-summit.svg';
+
+const GrowthSummit = () => {
+	return (
+		<Task
+			title="The WordPress.com Growth Summit"
+			description="Learn how to build and grow your site, from start to scale. Come join us for this exclusive two-day virtual event August 11-13."
```

Comment:
Please use `translate` as it's a good habit to stick to outside of example fragments. I understand we're targeting EN, though even that has US and GB variants.